bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/internal/handler/LGThinQBridgeHandler.java

```diff
@@ -0,0 +1,375 @@
+                }
+                lastDevicesDiscovered.put(deviceId, device);
+                lastDevicesDiscoveredCopy.remove(deviceId);
+            });
+            // the rest in lastDevicesDiscoveredCopy is not more registered in LG API. Remove from discovery
+            lastDevicesDiscoveredCopy.forEach((deviceId, device) -> {
+                logger.debug("LG Device '{}' removed.", deviceId);
+                lastDevicesDiscovered.remove(deviceId);
+
+                LGThinQAbstractDeviceHandler<? extends CapabilityDefinition, ? extends SnapshotDefinition> deviceThing = lGDeviceRegister
+                        .get(deviceId);
+                if (deviceThing != null) {
+                    deviceThing.onDeviceRemoved();
+                }
+                if (discoveryService != DUMMY_DISCOVERY_SERVICE && deviceThing != null) {
+                    discoveryService.removeLgDeviceDiscovery(device);
+                }
+            });
+
+            lGDeviceRegister.values().forEach(LGThinQAbstractDeviceHandler::refreshStatus);
+        }
+    }
+
+    @Override
+    public Collection<ConfigStatusMessage> getConfigStatus() {
+        List<ConfigStatusMessage> resultList = new ArrayList<>();
+        if (lgthinqConfig.username.isEmpty()) {
+            resultList.add(ConfigStatusMessage.Builder.error("USERNAME").withMessageKeySuffix("missing field")
+                    .withArguments("username").build());
+        }
+        if (lgthinqConfig.password.isEmpty()) {
+            resultList.add(ConfigStatusMessage.Builder.error("PASSWORD").withMessageKeySuffix("missing field")
+                    .withArguments("password").build());
+        }
+        if (lgthinqConfig.language.isEmpty()) {
+            resultList.add(ConfigStatusMessage.Builder.error("LANGUAGE").withMessageKeySuffix("missing field")
+                    .withArguments("language").build());
+        }
+        if (lgthinqConfig.country.isEmpty()) {
+            resultList.add(ConfigStatusMessage.Builder.error("COUNTRY").withMessageKeySuffix("missing field")
+                    .withArguments("country").build());
+
+        }
+        return resultList;
+    }
+
+    @Override
+
+    public void handleRemoval() {
+        ScheduledFuture<?> devicePollingJob = this.devicePollingJob;
+        if (devicePollingJob != null) {
+            devicePollingJob.cancel(true);
+        }
+        tokenManager.cleanupTokenRegistry(
+                Objects.requireNonNull(getBridge(), "Not expected bridge null here").getUID().getId());
+        super.handleRemoval();
+    }
+
+    @Override
+
+    public void dispose() {
+        ScheduledFuture<?> devicePollingJob = this.devicePollingJob;
+        if (devicePollingJob != null) {
+            devicePollingJob.cancel(true);
+            this.devicePollingJob = null;
+        }
+    }
+
+    @Override
+    public <T> T getConfigAs(Class<T> configurationClass) {
+        return super.getConfigAs(configurationClass);
+    }
+
```

Comment:
Why override if you don't change the implementation
```suggestion
```

Reply:
Good question :-D